README.md

```diff
@@ -63,6 +63,7 @@ export default () => (
   - [style](#style)
   - [visible (from Popover)](#visible)
   - [wrapperStyle](#wrapperStyle)
+  - [caretStyle](#caretStyle)
```

Comment:
Could this be added alongside `caret*` props?

Below `caretPosition` for example!

Please make sure to also change the position of the `caretStyle` paragraph :)